style/ruby/README.md

```diff
@@ -38,6 +38,9 @@ Ruby
 * Use heredocs for multi-line strings.
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
+* Break up method definitions over 80 characters long by
```

Comment:
I think it could be. 

Having the example seems valuable so that this case is covered clearly, but we may not need to actually list it in the list of styles here. 

This PR comes from me not knowing how we handle this, and then getting two differing answers in Slack, so it seems we do need to define this somehow.